flask_mongorest/resources.py

```diff
@@ -753,9 +761,28 @@ def get_skip_and_limit(self, params=None):
             if params.get('_skip') and int(params['_skip']) < 0:
                 raise ValidationError({'error': '_skip must be a non-negative integer (got "%s" instead).' % params['_skip']})
 
-            limit = min(int(params.get('_limit', self.default_limit)), max_limit)
+            cursor = params.get('_cursor')
+            if (params.get('_skip') or params.get('_limit')) and cursor:
+                raise ValidationError({'error': 'Cannot provide _cursor and _skip/_limit.'})
+
+            if cursor:
+                cursor_value = decode_sequential_cursor(cursor)
```

Comment:
Maybe `decode_sequential_cursor` can raise an exception if the cursor is invalid (malformed or expired).